### PR TITLE
fix(website): remove vague anchor do dont

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Locked Prettier version to 1.18.2
- [x] Removed second do/dont from `Anchor` doc page

Related issue: https://github.com/twilio-labs/paste/issues/109

### CODE REVIEW

File: packages/paste-website/src/pages/changelog/index.mdx [hunk 1]
```diff
@@ -4,7 +4,6 @@ title: Changelog
 
 import Changelog from '../../../CHANGELOG.md';
```
Comment (richbachman) [inline packages/paste-website/src/pages/changelog/index.mdx]: Removed this blank line for a test when trying to commit with newer Prettier versions.

File: package.json [hunk 2]
```diff
@@ -111,7 +111,7 @@
     "lerna": "^3.16.4",
     "lodash": "^4.17.14",
     "precise-commits": "^1.0.2",
-    "prettier": "1.19.1",
+    "prettier": "1.18.2",
```
Comment (TheSisb) [inline package.json]: What's the reason that we're pinning this version?

Comment (richbachman) [reply package.json]: There seems to be a bug in version 1.19.* that is blocking commits of `mdx` file changes.

Comment (richbachman) [reply package.json]: Prettier issue: https://github.com/prettier/prettier/issues/6943

Comment (TheSisb) [reply package.json]: Perfect, thanks!

**Pull request merged**